src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -53,24 +59,14 @@ export const transformIdentify: IdxStepTransformer = ({ formBag, widgetProps, tr
     options: { content: loc('primaryauth.title', 'login') },
   };
 
-  const identifierElement = getUIElementWithName(
-    'identifier',
-    uischema.elements as UISchemaElement[],
-  ) as FieldElement;
-
   // add username/identifier from config if provided
   if (identifierElement) {
+    // add username/identifier from config if provided
     if (username) {
-      identifierElement.options = {
-        ...identifierElement.options,
-        defaultOption: username,
-      };
+      data.identifier = username;
     } else if (features?.rememberMe && features?.rememberMyUsernameOnOIE) {
       const usernameCookie = getUsernameCookie();
-      identifierElement.options = {
-        ...identifierElement.options,
-        defaultOption: usernameCookie ?? undefined,
-      };
+      data.identifier = usernameCookie ?? undefined;
```

Comment:
Is the `??` necessary if the right side is going to be `undefined` anyways?

Reply:
Good catch, this is useless,  updating, ty!